### ISSUE TITLE
Rector crash with [ Fatal error: Uncaught Error: Call to undefined method Symfony\Component\Routing\Annotation\Route::getLocalizedPaths() ]

### ISSUE THREAD

**Issue by atompulse** [rectorphp/rector#3223]
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | 0.8.x-dev@e02410b             |
| Installed as   | docker rector/rector:latest                      |

Project code base "symfony/symfony": "3.4.12"
PHP version 7.1
Running with -vvv
Having more than 3k classes in /src

Whenever rector is being run and analyzes symfony controllers an error is being thrown:
```
[parsing] ../project/src/XXX/WebsiteBundle/Controller/YYY/ZZZ/DashboardController.php
Fatal error: Uncaught Error: Call to undefined method Symfony\Component\Routing\Annotation\Route::getLocalizedPaths() in /rector/packages/better-php-doc-parser/src/PhpDocNodeFactory/Symfony/SymfonyRoutePhpDocNodeFactory.php:41
Stack trace:
#0 /rector/packages/better-php-doc-parser/src/PhpDocParser/BetterPhpDocParser.php(195): Rector\BetterPhpDocParser\PhpDocNodeFactory\Symfony\SymfonyRoutePhpDocNodeFactory->createFromNodeAndTokens(Object(PhpParser\Node\Stmt\ClassMethod), Object(PHPStan\PhpDocParser\Parser\TokenIterator))
#1 /rector/packages/better-php-doc-parser/src/PhpDocParser/BetterPhpDocParser.php(169): Rector\BetterPhpDocParser\PhpDocParser\BetterPhpDocParser->parseTagValue(Object(PHPStan\PhpDocParser\Parser\TokenIterator), '@Route')
#2 /rector/vendor/phpstan/phpdoc-parser/src/Parser/PhpDocParser.php(54): Rector\BetterPhpDocParser\PhpDocParser\BetterPhpDocParser->parseTag(Object(PHPStan\PhpDocParser\Parser\TokenIterator))
#3 [internal function]: PHPStan\PhpDocParser\Parser\PhpDocParser->parseChild(Object(PHPStan\PhpD in /rector/packages/better-php-doc-parser/src/PhpDocNodeFactory/Symfony/SymfonyRoutePhpDocNodeFactory.php on line 41
```
The curious fact is that if I move this controller from /project/src/XXX/WebsiteBundle/Controller/YYY/ZZZ/
to 
/project/src/XXX/WebsiteBundle/Controller/YYY/
then the issue doesnt happen anymore although nothing has been changed in the controller code; might be that rector messes up some references internally.

At first impression this issue surfaces only when symfony controllers are being analyzed and are using annotations to define routes.

It's really weird because it doesnt affect all controllers, so if the same controller is moved to a more top level directory then the issue doesnt manifest, somehow only if the controller is nested into a more deeper directory this issue happens.

Important note: `vendor/symfony/symfony/src/Symfony/Component/Routing/Annotation/Route.php`
in project vendors does not have `getLocalizedPaths` method because this was introduced in symfony 4.1 but project uses symfony 3.4
BUT in https://github.com/rectorphp/rector/blob/c406fd3ab41dc0dc531058a52cdf39c2f7cab89b/packages/better-php-doc-parser/src/PhpDocNodeFactory/Symfony/SymfonyRoutePhpDocNodeFactory.php
```
return new SymfonyRouteTagValueNode(
            $route->getPath(),
            $route->getLocalizedPaths(),
            $route->getName(),
            $route->getMethods(),
            $route->getOptions(),
            $route->getDefaults(),
            $route->getHost(),
            $route->getRequirements(),
            $route->getCondition(),
            $annotationContent
        );
```
apparently `$route->getLocalizedPaths()` is being called without taking this into consideration.

So either rector should not use the project's symfony router OR should not call $route->getLocalizedPaths() when this method doesnt exists in project's current symfony version.

## Minimal PHP Code Causing Issue

https://getrector.org/demo/22379d97-9bfd-4bc4-9c72-5aea91bc38f4

## Expected Behaviour
Rector should not analyze or fail because of symfony route annotations since the chosen set should only be concerned with 'phpexcel-to-phpspreadsheet'

**Comment by atompulse** [comment]
@TomasVotruba could you please merge this PR?

**Comment by TomasVotruba** [comment]
Closed by #3224
